File: app/src/main/java/com/liushu/crazyandroid/ui/stage01/chapter02/activity/Demo020205Activity.java
package com.liushu.crazyandroid.ui.stage01.chapter02.activity;

import android.widget.ImageView;
import android.widget.TextView;

import com.jaydenxiao.common.base.BaseActivity;
import com.liushu.crazyandroid.R;

import butterknife.Bind;
import butterknife.OnClick;

public class Demo020205Activity extends BaseActivity {
    @Bind(R.id.iv_back)
    ImageView mIvBack;
    @Bind(R.id.tv_title_name)
    TextView mTvTitleName;

    @Override
    public int getLayoutId() {
        return R.layout.activity_demo020205;
    }

    @Override
    public void initPresenter() {

    }

    @Override
    public void initView() {
        mTvTitleName.setText("登录界面");

    }

    @OnClick(R.id.iv_back)
    public void onClick() {
        finish();
    }

   /* @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView();
    }*/
}

File: app/src/main/java/com/liushu/crazyandroid/ui/stage01/chapter02/activity/Demo020205Activity.kt
package com.liushu.crazyandroid.ui.stage01.chapter02.activity

import butterknife.OnClick
import com.jaydenxiao.common.base.BaseActivity
import com.liushu.crazyandroid.R
import kotlinx.android.synthetic.main.head_title_layout.*

class Demo020205Activity : BaseActivity() {

    override val layoutId: Int
        get() = R.layout.activity_demo020205


    override fun initView() {
        tv_title_name!!.text = "登录界面"

    }

    @OnClick(R.id.iv_back)
    fun onClick() {
        finish()
    }

    /* @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView();
    }*/
}
